playbooks/provisioning/openstack/server-setup.yaml

```diff
@@ -0,0 +1,72 @@
+---
+- hosts: cluster_hosts
+  become: true
+  pre_tasks:
+  - include: pre_tasks.yml
+  roles:
+  - role: hostnames
+
+# Subscribe Hosts
+- hosts: cluster_hosts
+  become: true
+  roles:
+  - role: subscription-manager
```

Comment:
agreed - I believe we need to have a bigger discussion around what components will move over from CASL to contrib. I'll add it to the agenda for this Friday's meeting.